package.json

```diff
@@ -22,7 +22,7 @@
     "react": "^15.4.1",
     "react-dom": "^15.4.1",
     "react-tabs": "^0.8.2",
-    "tachyons-egghead": "^1.1.5",
+    "tachyons-egghead": "eggheadio/tachyons-egghead#textfield",
```

Comment:
@trevordmiller I did `git push origin` instead of `git push upstream`... I should probably just remove `origin`!